backend/.env.example

```
@@ -36,7 +36,7 @@ SMTP_AUTH_PASS=<your-UTIA-password>
 # SMTP_PORT=465
 # SMTP_SECURE=true
 FROM_EMAIL=noreply@utia.cas.cz
-FROM_NAME="Cell Segmentation Platform"
+FROM_NAME="SpheroSeg"
```

dotenv-linter (3.3.0)

[warning] [QuoteCharacter] The value has quote characters (', ")

(QuoteCharacter)